db/seeds.rb

```diff
@@ -1 +1,89 @@
-User.create(username: 'Dice Master')
+# Users
+id = User.find_by(username: 'Dice Master').try(:id)
+
+user_1 = id ? User.find(id) : User.create!(username: 'Dice Master', 
+                                           password: 'Test_1234!',
+                                           password_confirmation: 'Test_1234!')
```

Comment:
Did it this way because Rails was having an issue with `find_or_create_by!` with this instance in that it only has a `password_digest` that gets handled by `BCrypt` and was erroring out saying `password` field didn't exist.

Reply:
oooh! look at that ternary operator!!